test/unittest/dds/participant/ParticipantTests.cpp

```diff
@@ -691,7 +699,7 @@ void set_server_qos(
 
 void set_environment_variable()
 {
-    std::string environment_servers = "84.22.253.128:8888;;localhost:1234";
+    std::string environment_servers = "84.22.253.128:8888;;UDPv4:[localhost]:1234;[2a02:ec80:600:ed1a::3]:8783";
```

Comment:
I would add a check that showed that using a single DNS entry both an IPv4 and an IPv6 locators are added, along with the required UDPv6 transport.

Reply:
And here it is: ParticipantTests.SimpleParticipantRemoteServerListConfigurationDNS